pkg/indexmanagement/converters.go

```diff
@@ -5,12 +5,13 @@ import (
 	"strconv"
 
 	apis "github.com/openshift/elasticsearch-operator/pkg/apis/logging/v1"
+	"github.com/openshift/elasticsearch-operator/pkg/constants"
 	"github.com/openshift/elasticsearch-operator/pkg/logger"
 )
 
 func calculateConditions(policy apis.IndexManagementPolicySpec, primaryShards int32) rolloverConditions {
-	// 40GB = 40960 1M messages
-	maxDoc := 40960 * primaryShards
+	// 40GB = 40960 1K messages
+	maxDoc := constants.TheoreticalShardMaxSizeInMB * 1000 * primaryShards
```

Comment:
1000 should also be a a named constant.
1K is 1024

Reply:
Yes, it could be like that.
This is back-port so let's fix it with next iteration.

Reply:
+1